Dockerfile

```
@@ -0,0 +1,15 @@
+FROM opensciencegrid/software-base:3.6-el7-release
```

```suggestion
FROM opensciencegrid/software-base:3.6-el8-release
```

We may as well use the latest and greatest OS available to us